public/stylesheet.css

```diff
@@ -0,0 +1,42 @@
+html { box-sizing: border-box; }
```

Comment:
I like how you used CSS to make Chitter look a little more interesting. It was nice to see the buttons for log in and out were aligned, and the timestamps on peeps were aligned, it looks very neat.